### PR TITLE
Poc/list cells

### PR DESCRIPTION
**Pull request by SilverGull**

### CODE REVIEW

File: src/kirby/components/list/list.component.tns.html [hunk 1]
```diff
@@ -5,10 +5,14 @@
     </FlexboxLayout>
   </ng-container>
   <RadListView [groupingFunction]="getSectionName" [items]="items" row="{{ rowNumberForListView(headerTemplate) }}">
-    <ng-template let-item="item" tkListItemTemplate>
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.tns.html]: Don't we need the `tkListItemTemplate` to have cell reuse?

I am not sure here as my nativescript knowledge is limited.

File: src/kirby/components/list/list.component.tns.scss [hunk 2]
```diff
@@ -4,20 +4,22 @@ $list-item-height: 44;
...
-.list-header-container {
-    padding-left: $list-item-padding;
-    padding-right: $list-item-padding;
-    height: $list-header-height;
+
+.row {
+    padding: 16 8 16 8;
+    justify-content: space-between;
+    flex-direction: row;
+//    height: $list-header-height;
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.tns.scss]: Is it needed if it is commented out?

File: src/kirby/components/list/list.component.tns.scss [hunk 3]
```diff
@@ -4,20 +4,22 @@ $list-item-height: 44;
...
-    color: get-color('primary');
-    padding-left: $list-item-padding;
-    padding-right: $list-item-padding;
-    padding-top: 5;
-    padding-bottom: 5;
-    border-bottom-width: 1;
-    border-bottom-color: #e7e7e7;
-}
\ No newline at end of file
+// .list-item-container {
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.tns.scss]: Is it needed if it is commented out?

File: src/kirby/components/list/list-cell/list-cell.component.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<!--
+    TODO: loose the div and use the host element
+   don't use ngStyle - maybe scss instead??? 
+   Same width for all cols
+
+-->
+<div class="cell" [ngStyle]="{'justify-content': alignVertical, 'align-items' : alignHorisontal}">
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list-cell/list-cell.component.html]: Looks like your editor is set to have tabs 4 spaces long and not 2 like the rest of the solution?

File: src/kirby/components/list/list-cell/list-cell.component.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
+.cell {
+    height: 100%;
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list-cell/list-cell.component.scss]: Looks like your editor is set to have tabs 4 spaces long and not 2 like the rest of the solution?

File: src/kirby/components/list/list-cell/list-cell.component.tns.html [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<FlexboxLayout class="cell" [justifyContent]="alignVertical" [alignItems]="alignHorisontal" [flexGrow]="flexGrow">
+    <Label  text="{{ lineOne }}" class="line-one" *ngIf="lineOne"></Label>
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list-cell/list-cell.component.tns.html]: Looks like your editor is set to have tabs 4 spaces long and not 2 like the rest of the solution?

File: src/kirby/components/list/list-cell/list-cell.component.tns.scss [hunk 7]
```diff
@@ -0,0 +1,16 @@
+.cell {
+    padding: 0 8 0 8;
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list-cell/list-cell.component.tns.scss]: Looks like your editor is set to have tabs 4 spaces long and not 2 like the rest of the solution?

File: src/app/examples/list-example/list-example.component.ts [hunk 8]
```diff
@@ -9,7 +9,7 @@ export class ListExampleComponent implements OnInit {
 
   myItems = [
     {title: 'Vestas Wind Systems', subTitle: '20 stk', amount: '18.309 DKK', detail: 225},
-    {title: 'A.P. Møller-Mærsk', subTitle: '2 stk', amount: '14.019 DKK', detail: -354},
+    {title: 'A.P. Møller-Mærsk', subTitle: '2 stk', amount: '14.019 DKK'},
```
[the remark this answers is not in the document]
Comment (SilverGull) [reply src/app/examples/list-example/list-example.component.ts]: It demonstrates how the row height adjusts to the number of lines in the cells. Comment added to code.

File: src/kirby/components/list/list.component.tns.html (hunk 1 above)
Comment (SilverGull) [reply src/kirby/components/list/list.component.tns.html]: I am not sure either. I have added it.

File: src/app/examples/list-example/list-example.component.ts [hunk 9]
```diff
@@ -9,6 +9,7 @@ export class ListExampleComponent implements OnInit {
 
   myItems = [
     {title: 'Vestas Wind Systems', subTitle: '20 stk', amount: '18.309 DKK', detail: 225},
+    // "detail" is missing to demonstrate how line height adjusts to the content
```
Comment (jonasjensen-brolstark) [inline src/app/examples/list-example/list-example.component.ts]: Could it be an idea to add the details property again. But maybe with 0 value and then make a check in the view code to not render that line of txt if the value is 0 ? 

That way we leave the data intact.

File: src/app/examples/list-example/list-example.component.html [hunk 10]
```diff
@@ -1,24 +1,29 @@
 <kirby-list [items]="myItems" (itemSelect)="onItemSelect($event)">
-  <kirby-list-cell *kirbyListCell="let account" [lineOne]="account.title" [lineTwo]="account.detail"></kirby-list-cell>
-  <kirby-list-cell *kirbyListCell="let acount" [lineOne]="acount.subTitle" alignHorisontal="flex-end"></kirby-list-cell>
-  <kirby-list-cell *kirbyListCell="let acount" [lineOne]="acount.amount" alignVertical="center" alignHorisontal="flex-end"></kirby-list-cell>
+  <kirby-list-cell 
+    *kirbyListCell="let account" 
+    [lineOne]="account.title" 
+    [lineTwo]="account.detail">
+  </kirby-list-cell>
+  <kirby-list-cell *kirbyListCell="let account" [lineOne]="account.subTitle" alignHorisontal="flex-end"></kirby-list-cell>
```
Comment (jonasjensen-brolstark) [inline src/app/examples/list-example/list-example.component.html]: Maybe you could just format all the `kirby-list-cell` elements like the first?

File: src/app/examples/list-example/list-example.component.ts (hunk 9 above)
Comment (SilverGull) [reply src/app/examples/list-example/list-example.component.ts]: not a bad idea

File: src/app/examples/list-example/list-example.component.html (hunk 10 above)
Comment (SilverGull) [reply src/app/examples/list-example/list-example.component.html]: prettier to the rescue

**Pull request merged**